guidelines/index.html

```diff
@@ -3539,11 +3535,15 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    
    <p>ある活動を完了させるために必要な利用者の一連の動作。</p>
    
-   <p class="example">ショッピングサイト上の一連のウェブページで目的を果たすためには、利用者が選択肢となりうる製品、価格及び内容を閲覧した後、製品を選択して注文し、配送先情報及び支払情報を入力する必要がある。
-   </p>
+   <aside class="example" id="example-15"><div class="marker">
+    <a class="self-link" href="#example-15">例<bdi> 15</bdi></a>
+    </div><p>ショッピングサイト上の一連のウェブページで目的を果たすためには、利用者が選択肢となりうる製品、価格及び内容を閲覧した後、製品を選択して注文し、配送先情報及び支払情報を入力する必要がある。
```

Comment:
細かい話なのですが、「入力」の部分は原文では "input" ではなく "provide" です。この文はユーザージャーニーの話をしていて、具体的なUIには触れていないものだと思いますので、「入力」より「提供」くらいのほうが良いように思います。
「注文」の部分は原文が "submit an order" で、order = 注文ですから全体としては「発注」というくらいでしょうか。
というところで以下のようにしてはどうかと思いました。
```suggestion
    </div><p>ショッピングサイト上の一連のウェブページで目的を果たすためには、利用者が選択肢となりうる製品、価格及び内容を閲覧した後、製品を選択して発注し、配送先情報及び支払情報を提供する必要がある。
```